### PR TITLE
[#52712] Adds front-end container

### PR DESCRIPTION
**Pull request by kimpepper**
(i.e. Go / Terraform / Docker)

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ARG ALPINE_VERSION=3.8
+
+FROM alpine:${ALPINE_VERSION}
+
+ENV PATH "$PATH:/data/node_modules/.bin"
+
+# Temporarily removing g++ to fix dns issue.
```
Comment (acbramley) [inline frontend/Dockerfile]: g++ is in the list so should this be removed?

**Comment by acbramley** [comment]
@kimpepper right but the comment doesn't line up with the packages being added

**Comment by nickschuch** [comment]
Fixed ;) Also added to the CircleCI builds (job was defined but not mapped)

**Comment by nicksantamaria** [comment]
I have thoughts - will make a PR against this branch for review.

**Comment by nicksantamaria** [comment]
Thoughts on #13 ?

**Pull request merged**